tests/integration/__tests__/patterns-dirs.js

```diff
@@ -153,6 +153,160 @@ if (path.sep === "/") {
   });
 }
 
+function isSymlinkSupported() {
+  if (process.platform !== "win32") {
+    return true;
+  }
+
+  const target = path.join(__dirname, "../cli/patterns-symlinks/file");
+  const symlink = path.join(__dirname, "../cli/patterns-symlinks/symlink");
+  fs.writeFileSync(target, "x");
+  try {
+    fs.symlinkSync(target, symlink);
+  } catch {
+    return false;
+  } finally {
+    fs.unlinkSync(target);
+  }
+  const stat = fs.lstatSync(symlink);
+  fs.unlinkSync(symlink);
+  return stat.isSymbolicLink();
+}
```

Comment:
Let's use a subdir to test. 
Since `test-*` already in `.gitignore`, we don't need clean. And if something goes wrong, them won't get committed either,

```suggestion
function isSymlinkSupported() {
  if (process.platform !== "win32") {
    return true;
  }

  const target = path.join(__dirname, "../cli/patterns-symlinks/test-symlink-feature-detect");
  fs.rmSync(target, { force: true, recursive: true });
  fs.mkdirSync(target);
  const symlink = path.join(target, "symlink");
  try {
    fs.symlinkSync(target, symlink);
  } catch {
    return false;
  }
  return fs.lstatSync(symlink).isSymbolicLink();
}
```

Reply:
Changed.